### PR TITLE
feat: add `new-release-git-tag` output variable

### PR DESCRIPTION
**Pull request by zachspar**
Adds `new-release-tag` output variable that is the _actual_ value of the `nextRelease.gitTag`.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ function verifyConditions() {
 function generateNotes(_pluginConfig, { nextRelease }) {
   core.setOutput("new-release-published", "true");
   core.setOutput("new-release-version", nextRelease.version);
+  core.setOutput("new-release-tag", nextRelease.gitTag);
```
Comment (felipecrs) [inline index.js]:
```suggestion
  core.setOutput("new-release-git-tag", nextRelease.gitTag);
```

For consistency

**Comment by felipecrs** [comment]
Please update https://github.com/felipecrs/semantic-release-export-data/blob/master/.github/workflows/ci.yaml to account for this new key.

**Comment by zachspar** [comment]
Thanks @felipecrs :)

**Review by felipecrs — APPROVED** [review]

**Comment by felipecrs** [comment]
> Thanks @felipecrs :) 

Thank you too! :)

**Pull request merged**